src/PhpSessionPersistence.php

```diff
@@ -16,7 +15,6 @@
 use Zend\Expressive\Session\Session;
 use Zend\Expressive\Session\SessionInterface;
 use Zend\Expressive\Session\SessionPersistenceInterface;
-
```

Comment:
Revert this change, we should have an empty line between different type of imports.